docs/cloud-runner/01-introduction.md

```diff
@@ -0,0 +1,70 @@
+# Introduction
+## Concept - What Does Cloud Runner Do
+
+**Cloud Runner enables you to send your project to run workflows on a Cloud Provider such as AWS or GCP, most commonly to build and test your project.** 
+
+**Primarily Cloud Runner is useful for game development because it supports large projects. Cloud Runner has first class support for the Unity game engine.**
+However, you could easily extend the system to run other tools and workloads. Cloud Runner can even use your projects to run workloads.
```

Comment:
This would more be an _addition_ than an _however._

However I would actually not keep this at the very start of the introduction but in another section at the end (e.g. advanced use cases)